### DISCUSSION TITLE
Lost Play Store

### DISCUSSION THREAD

**Discussion by troyboy27** [pgale/lineage_15.1_Installation_TSV#35] [Q&A] [answered]
I switched on a previously flashed (working but I didn't need it then), and there is no Google Play Store. TBH, I can't remember if it ever had it installed, so I wanted to know if, during the flashing, there was a step that I missed that installed the Play Store. Thanks for your insight!

**Comment by pgale** [comment, marked as answer]
You have to install it - it's all in the guide.

**Reply by troyboy27** [reply]
Can you add it after the flash, or do I need to start from scratch?

**Comment by troyboy27** [comment]
I can't believe I missed it out of the 3 I flashed. haha Thanks again @pgale for all your documentation and support to the community!

**Comment by troyboy27** [comment]
@pgale can you install the play store after it's been flashed, or do you need to complete the flashing again?

**Reply by pgale** [reply]
Not tried it but can't see why not - just go back to those steps and try it.
